tools_docker/test_docker/test_dockers.py

```diff
@@ -92,12 +93,34 @@ def test_default_file_created(tracim):
     assert tracim.file("/etc/uwsgi/apps-enabled/tracim_web.ini").is_symlink
 
     assert tracim.file("/etc/tracim/branding/color.json").is_file
-    assert tracim.file("/etc/tracim/branding/tracim-logo.png").is_file
-    assert tracim.file("/tracim/frontend/dist/assets/branding").is_directory
-
+    assert tracim.file("/etc/tracim/branding/manifest.json").is_file
+    assert tracim.file("/etc/tracim/branding/welcome-simple.html").is_file
+    assert tracim.file("/etc/tracim/branding/welcome-simple.css").is_file
+    assert tracim.file("/etc/tracim/branding/welcome-simple-text.html").is_file
+    assert tracim.file("/etc/tracim/branding/welcome-simple-bg.jpg").is_file
+    assert tracim.file("/etc/tracim/branding/images").is_directory
+    assert tracim.file("/etc/tracim/branding/images/tracim-logo.png").is_file
+    assert tracim.file("/etc/tracim/branding/images/safari-pinned-tab-icon.svg").is_file
+    assert tracim.file("/etc/tracim/branding/images/favicon").is_directory
+    assert tracim.file("/etc/tracim/branding/images/favicon/favicon.ico").is_file
```

Comment:
Can we check if file available here `frontend/dist/assets/branding.sample` are correctly available here `/etc/tracim/branding/` ?

For example new file added on develop in branding.sample but file not visible here (`incoming-call.ogg`) and also not tested.

Reply:
@inkhey do you think this is possible with `pytest-testinfra`?